app/helpers/taxonomy_helper.rb

```diff
@@ -138,13 +139,22 @@ def show_resource_if_allowed(f, taxonomy, resource_options)
     ids = "#{association.where(nil).klass.to_s.underscore.singularize}_ids".to_sym
 
     content_tag(:div, :id => resource, :class => "tab-pane") do
-      all_checkbox(f, resource) +
-      multiple_selects(f, association.where(nil).klass.to_s.underscore.pluralize.to_sym, association, taxonomy.selected_or_inherited_ids[ids],
-        {:disabled => taxonomy.used_and_selected_or_inherited_ids[ids],
-         :label => translated_label(resource, :select)},
-        {'data-mismatches' => taxonomy.need_to_be_selected_ids[ids].to_json,
-         'data-inheriteds' => taxonomy.inherited_ids[ids].to_json,
-         'data-useds' => taxonomy.used_ids[ids].to_json })
+      attr = association.where(nil).klass.to_s.underscore.pluralize.to_sym
+      selected_ids = taxonomy.selected_or_inherited_ids[ids]
+      opts = options.merge({:disabled => taxonomy.used_and_selected_or_inherited_ids[ids], :label => translated_label(resource, :select)})
+      html_opts = {'data-mismatches' => taxonomy.need_to_be_selected_ids[ids].to_json, 'data-inheriteds' => taxonomy.inherited_ids[ids].to_json,
+                   'data-useds' => taxonomy.used_ids[ids].to_json,
+                   'disabled' => (taxonomy.ignore_types.any? resource.to_s.singularize.capitalize)
+      }
```

Comment:
The `html_options` are not used in the new component. Could we maybe not do `to_json` just for parsing it from Json again?
```suggestion
      mismatches = taxonomy.need_to_by_selected_ids[ids]
      inheriteds = ...
      useds = ....
```